src/vm/i386/excepx86.cpp

```diff
@@ -38,6 +38,7 @@
 #include "asmconstants.h"
 #include "virtualcallstub.h"
 
+#ifndef WIN64EXCEPTIONS
```

Comment:
This whole file should be excluded for WIN64EXCEPTIONS and so you can revert all the changes in it.